client/src/App.tsx

```diff
@@ -18,47 +18,55 @@ import Messages from './pages/Messages/Messages';
 import ProtectedRoute from './components/ProtectedRoute';
 import Unauthorized from './pages/Unauthorize/Unauthorized';
 import SitterOwner from './pages/Sitters/SitterOwner';
+import { Elements } from '@stripe/react-stripe-js';
+import { loadStripe } from '@stripe/stripe-js';
 
 import './App.css';
 
+const stripePromise = loadStripe(
+  'pk_test_51J5B2eKVTqeqLi3vW1fqFgTEllljxwuWTGNwdOHOfzFZfSJeGlYisgmxohIEQLFUuNsefjYWrEtYr6errTkZfbq200PVHMyOyJ',
```

Comment:
Please store API key as an environment variable before merging.

Reply:
I was going to store it as an environment variable before I created a PR, but I found this on the stripe site
https://stripe.com/docs/keys
```
There are also two types of API keys: publishable and secret.

Publishable API keys are meant solely to identify your account with Stripe, they aren’t secret. In other words, they can safely be published in places like your Stripe.js JavaScript code, or in an Android or iPhone app.
Secret API keys should be kept confidential and only stored on your own servers. Your account’s secret API key can perform any API request to Stripe without restriction.

Reply:
Thanks for sharing this resource!